package.json

```diff
@@ -14,6 +14,7 @@
   },
   "scripts": {
     "dist": "webpack && webpack --config=webpack.config.minified.js",
+    "lib": "rm -rf ./lib && babel ./modules -d lib && rm -rf ./lib/__tests__ ./lib/__mocks__",
```

Comment:
use rimraf instead of rm -rf so it works on windows (not sure if there are other blockers, but no need to add new ones)